### PR TITLE
Separate step 2 script for ami

### PR DESCRIPTION
**Pull request by karimi**
This PR creates a separate `bootstrap_step-2` for use in creation of AWS instances from AMI.
`bootstrap_step-2` after completion writes the `RhinoComputeUser` password to the console. In an AMI situation, the script is run headlessly when instance is created from the AMI, the user does not see the console.I'm proposing writing the output to a text file on the desktop.

### CODE REVIEW

File: script/production/module_iis_configure_ami.ps1 [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+cmd /c icacls $rhinoComputePath /grant ("IIS AppPool\$appPoolName" + ':(OI)(CI)F') /t /c /q 
+cmd /c icacls $computeGeometryPath /grant ("IIS AppPool\$appPoolName"+ ':(OI)(CI)F') /t /c /q 
+
+Write-Step "Setting environment variable for log paths"
+SetEnvVar 'RHINO_COMPUTE_LOG_PATH' "$rhinoComputePath\logs"
+
+Write-Step "Starting rhino.compute site" 
+Start-IISSite -Name $websiteName
+
+Start-Transcript -Path C:\Users\Administrator\Desktop\ReadMe.txt
```
Comment (andyopayne) [inline script/production/module_iis_configure_ami.ps1]: I think this line needs to change to:
Start-Transcript -Path $env:USERPROFILE\Desktop\ReadMe.txt
The userprofile might not always be "Administrator". This is usually the case for AWS VMs, but it's easy to set the admin user profile name in Azure when setting up a VM, so I think the text above will always pull the current logged in user's desktop profile.

File: script/production/module_iis_configure_ami.ps1 [hunk 2]
```diff
@@ -60,6 +60,10 @@ function New-RandomPassword {
 }
 #EndRegion funcs
 
+Write-Host "Generating new API Key"
```
Comment (andyopayne) [inline script/production/module_iis_configure_ami.ps1]: If you're creating you're own API Key here, then you need to remove the requirement for the user to pass in their API Key at the beginning of the first step. If you look at module_rhino.ps1, you'll notice at the very beginning that it asks the user to pass in their Email, their Auth token, and their API Key. At the very least, the API key requirement should be removed from that script.
And given the fact that it looks like you'll also be changing the step 1 part of the bootstrap script... My recommendation would be to create a new folder directory called "AMI" that sits next to the "production" directory. In the AMI folder, we should copy all the necessary files to run the bootstrap script (all steps). That way we leave the current bootstrap script directory alone, and simply have a new directory for this new setup.

**Review by andyopayne — CHANGES_REQUESTED** [review]
Please see my comments about rearranging the folder structure to create a new AMI directory and all scripts would go in there... as well as removing the API key requirement from the rhino module as well as using the environment variable to pull the current user's desktop path location.

File: script/production/module_iis_configure_ami.ps1 (hunk 1 above)
Comment (karimi) [reply script/production/module_iis_configure_ami.ps1]: That makes sense

File: script/production/module_iis_configure_ami.ps1 (hunk 2 above)
Comment (karimi) [reply script/production/module_iis_configure_ami.ps1]: Done

**Pull request merged**